Automattic/jetpack
# Add Stats counts (highlights) logic to My Jetpack

## Proposed changes:

* Add the StatsSection component to My Jetpack page
* Add logic to only display the section if Stats module is active
* Connect Stats section to the store (and backend using the endpoints from Odyssey Stats)

<img width="1704" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/60f8d85e-e7cf-43e7-ab99-7fc4d4913f8e">


### Other information:


## Jetpack product discussion



pbNhbs-80w-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:

NOTE: for now, the base branch is the one used in #33160. I'll do a `git rebase` as soon as the other PR is merge, and this one will be much cleaner.







* Point your site to this branch using Jetpack Beta or run this branch locally
* Visit `/wp-admin/admin.php?page=my-jetpack` with Stats active, and you'll see the new Stats section on this page
* Visit `/wp-admin/admin.php?page=jetpack_modules` and disable the Jetpack Stats module
* Go back to My Jetpack and confirm the section isn't there anymore, and there's no call to the API (`/jetpack/v4/stats-app/sites/{site_id}/stats/highlights?_locale=user`)
* With the module enabled, in the Stats section on My Jetpack, click on "See detailed stats". Confirm that it redirects to the Jetpack Stats page
